modules/37_industry/subsectors/datainput.gms

```diff
@@ -144,17 +144,20 @@ loop (industry_ue_calibration_target_dyn37(out),
     );
   );
 
-  loop (regi_group(ext_regi,regi)$( 
-                     smax(ttot, p37_energy_limit_def(ttot,ext_regi,out)) ne 0 ),
+  loop ((ext_regi,all_regi(regi))$(
+                (sameas(all_regi,ext_regi) OR regi_group(ext_regi,all_regi))
```

Comment:
Is there a specific reason to use all_regi here instead of regi. I think they should usually be equal.

Reply:
No.  That was left over.